dashboard/db/migrate/20170413194726_replace_placeholder_csp_workshop_subjects.rb

```diff
@@ -0,0 +1,23 @@
+class ReplacePlaceholderCspWorkshopSubjects < ActiveRecord::Migration[5.0]
+  SUBJECT_NAMES = [
+    {from: 'Summer Workshop', to: '5-day Summer'},
```

Comment:
I need it to work in both directions in case we need to roll back. Which value from of the constants would be used? I find this easier to deal with and clear to read. While I generally don't like hardcoded strings in reusable code, for a migration like this I actually prefer it.

Also the general guidance is not to rely on logic from the models in migrations for if they need to be run again and the model is in a different iteration.